wmcore_base/ContainerScripts/installWMCorePy3.sh

```diff
@@ -0,0 +1,16 @@
+#! /bin/sh
+
+# Download the script to install everything
+
+# Swap comments after merging https://github.com/dmwm/WMCore/pull/10496
+# curl https://raw.githubusercontent.com/dmwm/WMCore/master/test/deploy/deploy_unittest_py3.sh > /home/dmwm/ContainerScripts/deploy_unittest_py3.sh
+curl https://raw.githubusercontent.com/mapellidario/WMCore/10487-fix/test/deploy/deploy_unittest_py3.sh > /home/dmwm/ContainerScripts/deploy_unittest_py3.sh
```

Comment:
Just a reminder to get this updated before merged.

Reply:
I have changed this! Being validated in [this pipeline](https://gitlab.cern.ch/dmapelli/dmwm/-/pipelines/2632252)